http/middleware.go

```diff
@@ -13,27 +14,50 @@ import (
 // Middleware represents a func that chains http handlers
 type Middleware func(next http.Handler) http.Handler
 
+// Used with logging handlers. Returns true if the request should NOT be logged.
+type LogFilter func(r *http.Request) bool
+
 // DefaultRequestLogger provides a default middleware chain with
 // AccessHandler and RequestIDHandler middlewares
 //
 // Example:
-// 	DefaultRequestLogger(log, "requestid", "Request-Id")(handler)
+//
+//	DefaultRequestLogger(log, "requestid", "Request-Id")(handler)
 var DefaultRequestLogger = func(log zerolog.Logger, fieldKey, headerName string) Middleware {
 	return func(next http.Handler) http.Handler {
-		return hlog.NewHandler(log)(AccessHandler(RequestIDHandler(fieldKey, headerName)(next)))
+		return hlog.NewHandler(log)(
```

Comment:
Might be worth testing the DefaultRequestLogger or atleast the LogFilter function so we know it's working as expected